TheaterSchedule.BLL/Infrastructure/ErrorDetails.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace TheaterSchedule.Infrastructure
+{
+    public class ErrorDetails
+    {
+        public int StatusCode { get; set; }
+        public string Message { get; set; }
+
+        public override string ToString()
+        {
+            return JsonConvert.SerializeObject(this);
```

Comment:
don't mix string representation of object and serialization
ToString is mentioned to be some user friendly representation of object
https://docs.microsoft.com/en-us/dotnet/api/system.object.tostring?view=netframework-4.7.2
```
Overrides of the ToString() method should follow these guidelines:
The returned string should be friendly and readable by humans.
The returned string should uniquely identify the value of the object instance.
The returned string should be as short as possible so that it is suitable for display by a debugger.

```
So it's better implement toString as human readable version of object and add attribute Serializable for passing it via network


Object.ToString is the major formatting method in the .NET Framework. It converts an object to its string representation so that it is suitable for display.